components/primitives/Dialog.tsx

```diff
@@ -118,7 +118,7 @@ const Dialog = forwardRef<
         <AnimatePresence>
           {open && (
             <DialogPrimitive.DialogPortal forceMount {...portalProps}>
-              <AnimatedOverlay style={{ opacity: 1 }} {...overlayProps} />
+              <AnimatedOverlay style={{ opacity: 0.6 }} {...overlayProps} />
```

Comment:
Does it make sense to change this globally or should we just pass it into the overlayProps?